### PR TITLE
AB#834 merge all language files

### PR DESCRIPTION
**Pull request by PeculiarGoat**
Does what it says on the tin.

### CODE REVIEW

**Review by stajs — APPROVED** [review]
Now that we are getting a few of these files, is it worth grouping under an `i18n` folder?

File: package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
     "css:watch:local:variables": "chokidar local/site/styles/_variables.scss -c 'npm run css:local:variables'",
     "css:watch:local:custom": "chokidar local/site/styles/*.scss -i local/site/styles/_variables.scss -c 'npm run css:local:custom'",
     "merge:json:kibble": "node tools/merge-json.js output/kibble.json core/kibble.json local/kibble.json",
-    "merge:json:lang:AU": "node tools/merge-json.js output/site/en_AU.all.json core/site/en_AU.all.json local/site/en_AU.all.json",
-    "json:kibble:lang:watch": "chokidar local/site/en_AU.all.json -c 'npm run merge:json:lang:AU'"
+    "merge:json:lang": "for file in local/site/**_**.all.json; do filename=$(echo $file | grep -Eo '[^/]+$'); node tools/merge-json.js output/site/$filename core/site/$filename local/site/$filename; done",
```
Comment (stajs) [inline package.json:31]: Is double-`*` redundant, i.e. would `*_*.all.json` match?

Comment (PeculiarGoat) [reply package.json:31]: Ah yeah probably.

**Comment by PeculiarGoat** [comment]
> Now that we are getting a few of these files, is it worth grouping under an `i18n` folder?

Probably a good idea!  I guess that would need changes in core-template too.

**Pull request merged**